configure

```diff
@@ -2841,34 +2841,35 @@ case $_host_os in
 		append_var LIBS "-lcitro3d"
 		;;
 	amigaos*)
-		# Leave out resources by default to save binary size
-		_builtin_resources=no
 		# AmigaOS (PPC) target
 		_port_mk="backends/platform/sdl/amigaos/amigaos.mk"
 		add_line_to_config_mk 'AMIGAOS = 1'
-		append_var CXXFLAGS "-mlongcall"
```

Comment:
This flag has been completely removed - it is used to produce smaller executables. Is it not needed anymore?